test/runtests.jl

```diff
@@ -31,7 +31,9 @@ const is_CI = haskey(ENV,"CI")
   if GROUP == "All" || GROUP == "NNRODE"
         @time @safetestset "NNRODE" begin include("NNRODE_tests.jl") end
   end
-
+  if GROUP == "All" || GROUP == "NNPKOLMOGOROV"
+        @time @safetestset "NNParamKolmogorov" begin include("NNParamKolmogorov_tests.jl") end
```

Comment:
How long does this test set take? I don't think it needs to be separated from NNRODE. But if it does, it needs to be added to the Github CI action.

Reply:
Just in the order of minutes I guess. Yeah we can add it with NNRODE. I ll add this one and check,